experiments/list_unsupported.sh

```diff
@@ -61,19 +61,33 @@ if ! (gnat --version | grep -q 'GNAT GPL 2016') ; then
    exit 5
 fi
 
+# Sane ADA_INCLUDE_PATH and GPR_PROJECT_PATH?
 GNAT2GOTO=`command -v gnat2goto`
 ADA_HOME=`command -v gnat`
-ADA_HOME="$(dirname "$ADA_HOME")/.."
+ADA_HOME=$(cd "$(dirname ${ADA_HOME})/.." 2>/dev/null && pwd)
 PLATFORM=`${ADA_HOME}/bin/gcc -dumpmachine`
 DEF_ADA_INCLUDE_PATH="${ADA_HOME}/lib/gcc/${PLATFORM}"
 ADA_GCC_VERSION=`${ADA_HOME}/bin/gcc -dumpversion`
 DEF_ADA_INCLUDE_PATH="${DEF_ADA_INCLUDE_PATH}/${ADA_GCC_VERSION}/rts-native/adainclude:${ADA_HOME}/include"
 export ADA_INCLUDE_PATH="${ADA_INCLUDE_PATH:-$DEF_ADA_INCLUDE_PATH}"
 
-export GPR_PROJECT_PATH="${GPR_PROJECT_PATH:-/opt/gnat/lib/gnat}"
-if [ ! -d "$GPR_PROJECT_PATH" ]; then
-   echo >&2 "GPR project path does not exists!"
+if [ -n "$GPR_PROJECT_PATH" -a ! -d "$GPR_PROJECT_PATH" ]; then
```

Comment:
FYI  `GPR_PROJECT_PATH` like many other path variables doesn't necessarily consist of only a single path, so this check actually excludes some possibly valid configurations. Not sure if that's ever going to come up, but it'll be a nuisance if it does.

Reply:
Ah yes - I'd thought about this in an earlier cleanup but then did nothing about it. Again, for the moment I think the assumption of a single directory is OK for the purpose we have in mind.